app/src/main/java/com/oracle/javafx/scenebuilder/app/menubar/MenuBarController.java

```diff
@@ -1171,6 +1176,24 @@ public String getTitle() {
         insertMenu.setOnMenuValidation(onCustomPartOfInsertMenuValidationHandler);
         
         windowMenu.setOnMenuValidation(onWindowMenuValidationHandler);
+
+        // Modify the Check Updates menu item if there is an update available.
+        // Icons by Font Awesome (https://fontawesome.com/license/free) under CC BY 4.0 License
+        AppSettings.getLatestVersion(latestVersion -> {
+            var updateAvailable = AppSettings.isUpdateAvailable();
+            Image icon = new Image(MenuBarController.class
+                    .getResource(updateAvailable ? "download_icon.png" : "check_icon.png")
```

Comment:
I think we can remove `warn_icon` from this PR.

Reply:
Done. Good catch. :-)